systemvm/patches/debian/config/opt/cloud/bin/cs/CsFile.py

```diff
@@ -114,7 +114,10 @@ def greplace(self, search, replace):
 
     def search(self, search, replace):
         found = False
-        logging.debug("Searching for %s and replacing with %s" % (search, replace))
+        replace_filtered = replace
+        if re.search("PSK \"", replace):
+            replace_filtered = re.sub(r'".*"', '"****"', replace)
+        logging.debug("Searching for %s and replacing with %s" % (search, replace_filtered))
```

Comment:
"Searching for %s and replacing the value following it with %s" would be closer to the truth